packages/tests/tests/light-push/index.node.spec.ts

```diff
@@ -19,7 +19,7 @@ import {
   TestEncoder
 } from "./utils.js";
 
-describe("Waku Light Push", function () {
+describe.only("Waku Light Push", function () {
```

Comment:
@fbarbu15 !!!

Reply:
Thanks, will keep an eye out for this in the future
This one was removed imediately after with this [commit ](https://github.com/waku-org/js-waku/pull/1645/commits/2b7fb396d3b37a70b5e5636da5c0f10a1e7915a9)